packages/tina-graphql/src/index.ts

```diff
@@ -98,7 +98,7 @@ export const githubRoute = async ({
     owner,
     repo,
     ref: branch,
-    cache: simpleCache,
+    cache: s3Cache,
```

Comment:
I think this should be passed in as an override to the `simpleCache` for simplicity. So you'd add an argument to the `githubRoute` params:
```ts
  return githubRoute({
    accessToken: authentication.token,
    owner,
    repo,
    query: requestBody.query,
    variables: requestBody.variables,
    branch,
    cache: s3Cache
  });
```
This way we can just pass it from where it's called [here](https://github.com/tinacms/teams-serverless/blob/main/js/content-api/src/graphql-content.ts#L105-L110) and possibly move the S3 logic to that other repo